.github/actions/checkout_ancestor_commit/action.yaml

```diff
@@ -0,0 +1,63 @@
+name: "Checkout Ancestor Commit"
+description: >-
+  Checksout the most recent common ancestor of two commits.
+
+  This action clones all necessary repositories, no prior clone required.
+
+inputs:
+  ref:
+    description: >-
+      The reference for which the ancestor commit, with the current checked out
+      commit, should be found.
+    required: false
+    default: master
+  repository:
+    description: >-
+      Repository name with owner of the reference commit. For example,
```

Comment:
Really nitpicking here but `repository_owner/repository_name` of the repository containing the reference commit :P